m/msgpack-c/msgpack-c_ubi_8.3.sh

```diff
@@ -0,0 +1,67 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : msgpack-c
+# Version       : 3.2.1                                                                                                               
+# Source repo   : https://github.com/msgpack/msgpack-c 
+# Tested on     : UBI 8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Siddhesh Ghadi <Siddhesh.Ghadi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+set -ex 
+
+dnf update -y
+
+#adding extra repo 
+dnf -y --disableplugin=subscription-manager install \
+    http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \
+    http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/Packages/centos-linux-repos-8-2.el8.noarch.rpm \
+    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+#installing dependency 
+dnf install gcc gcc-c++ make cmake libtool git -y
+dnf install zlib-devel python3 flex bison wget unzip  -y 
+
+alternatives --set python /usr/bin/python3 
+
+# getting sources 
+cd ~
+git clone https://github.com/msgpack/msgpack-c --branch cpp-3.2.1                                                                                                               
```

Comment:
Can we made cpp-3.2.1 as a variable and can be set from Parameters.
Something like:
`VERSION?=cpp-3.2.1`

Even for the other versions (google-test-release-version)

Reply:
Updated the script with requested changes. @kandarpamalipeddi

Reply:
@Siddhesh-Ghadi 
Sorry for the confusion with the earlier comment.
Can you make it as parameters or command line arguments.

Instead of `VERSION=${VERSION:-'cpp-3.2.1'}` something like `VERSION=${1:-'cpp-3.2.1'}`
If that doesn't work,
```
VERSION=$1
if [ -z $VERSION ]
then
  VERSION=cpp-3.2.1
fi
```
Also, squash your commits once you are done with these changes

Thanks in advance.

Reply:
`VERSION=${1:-'cpp-3.2.1'}` works.